### PR TITLE
docs: Updated Email.js provider documentation page

### PR DESCRIPTION
**Pull request by akarsh-jain-790**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Fixes: [[NV-909] - Create a Email.js provider documentation page ](https://github.com/novuhq/novu/issues/1665)

### CODE REVIEW

File: docs/docs/channels/email/emailjs.md [hunk 1]
```diff
@@ -1,23 +1,27 @@
-# EmailJS
+# Email.js
 
-[EmailJS](https://www.emailjs.com/) helps to send emails using client-side technologies only. No server is required – just connect EmailJS to one of the supported email services, create an email template, and use our Javascript library to trigger an email. You can use EmailJS to send the content of a form filled by your users directly to your email address, using a simple connection provided by EmailJS.
+You can use the [Email.js](https://www.emailjs.com/) provider to send transactional emails to your customers using the Novu Platform with a single API to create multi-channel experiences.
```
Comment (p-fernandez) [inline docs/docs/channels/email/emailjs.md:3]: I am sorry but I don't find this is an improvement to the existing text.

File: docs/docs/channels/email/emailjs.md [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
 
-## How to connect a form to your email?
+## Find the API Key
 
-- Create a simple HTML form.
-- Create an account on EmailJS and verify your email.
-- Create a new service and do all the settings.
-- Create a connection using CDN or API.
+To find your Email.js API key, log into your Email.js account and navigate to the [API Keys](https://dashboard.emailjs.com/admin/account) page.
+It is suggested that you create a new API key for use with Novu. To successfully send emails, you will need to add the following permissions to your API key:
```
Comment (p-fernandez) [inline docs/docs/channels/email/emailjs.md:12]: This bit doesn't make sense. It is incomplete.
Also I couldn't find any recommendation from Email.js to create a different API key for different consumers from the same owner.

File: docs/docs/channels/email/emailjs.md [hunk 3]
```diff
@@ -1,23 +1,27 @@
...
 
-## Integrate EmailJS with Novu
+## Authenticate your sender identity
+
+All email services require some sort of authentication to send the emails on your behalf. That makes it a bad idea to use them directly from client-side – revealing your password or your secret keys will allow anyone to send emails on your behalf.
+
+Email.js keeps your authentication details on the server-side, and the client-side code just triggers a predefined email template, similarly to how any client-server application is working.
```
Comment (p-fernandez) [inline docs/docs/channels/email/emailjs.md:18]: This bit belongs to this area: https://www.emailjs.com/docs/faq/cant-i-use-services-like-sendgrid-or-mandrill-directly/
It tries to explain the benefits of using Email.js when dealing with other services. To be honest, has little relation to the purpose of this documentation that is to teach to the users how to integrate this provider in Novu.

**Review by p-fernandez — CHANGES_REQUESTED** [review]

**Comment by scopsy** [comment]
Closing this one due to in activity 🙏

**Pull request closed without merging**